crates/wizer/src/lib.rs

```diff
@@ -48,11 +53,8 @@ const DEFAULT_KEEP_INIT_FUNC: bool = false;
 pub struct Wizer {
     /// The Wasm export name of the function that should be executed to
     /// initialize the Wasm module.
-    #[cfg_attr(
-        feature = "clap",
-        arg(short = 'f', long, default_value = "wizer.initialize")
-    )]
-    init_func: String,
```

Comment:
Can we continue to have a default value here, since it shows up in the help text, but just make it the same for components and core modules? It can't be `wizer.initialize` anymore because the `.` means it isn't a valid component export name. And `component-init` doesn't make sense for core modules. How about `wizer-init` or `wizer-initialize`?